From 7a2ccdc37863887eda7ab479088844207a2bc54f Mon Sep 17 00:00:00 2001
From: legend <kygx.legend@gmail.com>
Date: Fri, 6 Jan 2017 13:51:19 +0800
Subject: [PATCH] [GLOG] Customize log format for Husky

---
 src/logging.cc | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/logging.cc b/src/logging.cc
index 0b5e6ee..b50d304 100644
--- a/src/logging.cc
+++ b/src/logging.cc
@@ -1237,8 +1237,9 @@ void LogMessage::Init(const char* file,
   //    (log level, GMT month, date, time, thread_id, file basename, line)
   // We exclude the thread_id for the default thread.
   if (FLAGS_log_prefix && (line != kNoLogPrefix)) {
-    stream() << LogSeverityNames[severity][0]
-             << setw(2) << 1+data_->tm_time_.tm_mon
+    stream() << "[" << LogSeverityNames[severity] << ' '
+             << setw(4) << 1900+data_->tm_time_.tm_year << '-'
+             << setw(2) << 1+data_->tm_time_.tm_mon << '-'
              << setw(2) << data_->tm_time_.tm_mday
              << ' '
              << setw(2) << data_->tm_time_.tm_hour  << ':'
--
2.7.4

